plugins/ci/skills/reevaluate-job-runs/reevaluate_job_runs.py

```
@@ -0,0 +1,165 @@
+                last_err = "gateway returned an HTML error page (likely 504 timeout)"
+            else:
+                try:
+                    return json.loads(body).get("results", []), None, False
+                except ValueError:
+                    last_err = "server returned a non-JSON response body"
+        except urllib.error.HTTPError as e:
+            detail = ""
+            try:
+                detail = e.read().decode("utf-8")
+            except Exception:
+                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)